tests/dnssd-tests.sh

```diff
@@ -12,8 +12,18 @@
 #   ./dnssd-tests.sh "Printer DNS-SD Service Instance Name"
 #
 
+# Currently unused
```

Comment:
How about using an environment variable for this instead? One advantage is that it would be compatible with the Windows tools as well, and another is that it avoids potential issues with spaces in the argument names (which the below code doesn't put in quotes...)